src/lib.rs

```diff
@@ -220,13 +291,17 @@ mod tests {
     fn test_roundtrips() {
         let f = Fernet::new(&base64::encode_config(&vec![0; 32], base64::URL_SAFE));
 
-        for val in [
-            b"".to_vec(),
-            b"Abc".to_vec(),
-            b"\x00\xFF\x00\x00".to_vec(),
-        ].into_iter() {
+        for val in [b"".to_vec(), b"Abc".to_vec(), b"\x00\xFF\x00\x00".to_vec()].into_iter() {
             assert_eq!(f.decrypt(&f.encrypt(&val)), Ok(val.clone()));
         }
     }
 
+    #[test]
+    fn test_multi_roundtrips() {
+        let f = MultiFernet::new(vec![base64::encode_config(&vec![0; 32], base64::URL_SAFE)]);
+
+        for val in [b"".to_vec(), b"Abc".to_vec(), b"\x00\xFF\x00\x00".to_vec()].into_iter() {
+            assert_eq!(f.decrypt(&f.encrypt(&val)), Ok(val.clone()));
+        }
+    }
```

Comment:
More tests needed: https://github.com/pyca/cryptography/blob/master/tests/test_fernet.py#L134 contains examples of the types of tests I'd like to see

Reply:
Added the other tests. Didn't add the rotate tests as the TTL stuff isn't there yet.